src/settings.py

```diff
@@ -6,6 +6,22 @@
 import botconfig
 
 LANGUAGE = 'en'
+
+## TLS settings
+# If SSL_CERTFP is supplied, the bot will attempt to verify that with the server. If not, then the
+# bot will attempt certificate verification, otherwise it will abort the connection.
+# You may specify the hash algorithm to use when verifying fingerprints. If unspecified, it will
+# attempt to autodetect the hash type of each fingerprint individually. (Limited to MD5, SHA1,
+# SHA224, SHA384, and SHA512).
+# Multiple fingerprints may be comma separated for networks with multipleservers. Check your Python
+# version for suported algorithms.
+# Syntax: SSL_CERTFP = "[HASH-ALGO:]fingerprint1,fingerprint2,fingerprint3"
```

Comment:
This is dumb. Make SSL_CERTFP be a set of fingerprint strings instead of a comma-separated list. There is no need to specify hash algo explicitly as we can determine that based on string length. The default should be set() instead of None.

Reply:
@skizzerz Determining the hash algorithm based on length is not reliable so it needs a spot to be explicitly notated. Doesn't matter to me how it is done. It could be moved into a `SSL_CERTFP_METHOD` string and the fingerprints could be made as a set. The code will "auto-determine" common hash algorithms but if one of the more exotic hashes are used then it needs to be explicitly set.

As of Python3.6, Blake2 is supported which is a variable length hash function. Additionally,  SHA3-256 and SHA2-256 use the same length.

Reply:
I see no point in supporting every hash algo that python supports. Pick a small handful that will be most commonly used in IRC and only support that subset. For example, I see no reason to support SHA2-224 or SHA2-384.